pkg/controller/backup_controller.go

```diff
@@ -249,7 +263,6 @@ func (b *backupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		request.Status.Phase = velerov1api.BackupPhaseInProgress
 		request.Status.StartTimestamp = &metav1.Time{Time: b.clock.Now()}
 	}
-
```

Comment:
To minimize change from upstream since this is in our fork, lets not include whitespace changes like this.